tools-ui/src/main/java/com/hedera/hashgraph/client/ui/popups/KeyDesignerPopup.java

```diff
@@ -118,6 +118,8 @@
 	public static final String REMOVE_BUTTON_STYLE =
 			"-fx-background-color: transparent; -fx-text-fill: red;-fx-border-color: indianred; " +
 					"-fx-border-radius: 5";
+	public static final String CANNOT_UPDATE_THRESHOLD =
+			"The selected threshold does not have a list of keys associated with it. The threshold cannot be updated";
```

Comment:
As this field is not shared to any other class, it should be `private`. If it has to be shared, consider package scope before making it `public`. The same guideline applies to fields above this one that you did not touch.

Reply:
makes sense. I will make the change